parachain/primitives/router/src/inbound/mod.rs

```diff
@@ -247,15 +248,15 @@ where
 	fn convert_location(location: &MultiLocation) -> Option<AccountId> {
 		if let MultiLocation { interior: X1(GlobalConsensus(Ethereum { chain_id })), .. } = location
```

Comment:
Should we take into account the `parents` when generating sovereign accounts?

Reply:
For consensus systems on Ethereum, no. Whenever there is the `GlobalConsensus` in the location, you can safely ignore `parents` (according to Adrian/Branislav).